tutorials/setup.md

```diff
@@ -130,6 +130,63 @@ make BUILDTAGS='seccomp apparmor'
 | exclude_graphdriver_overlay      | exclude overlay as a storage option             | <none>       |
 | ostree                           | build storage using ostree                      | ostree       |
 
+## Static builds
+
+It is possible to build a statically linked binary of CRI-O by using the
+provided [nix](https://nixos.org/nix) package [within this repository](../nix).
```

Comment:
this "within this repository" link gives me a 404

Reply:
you'll want (/nix) I think @saschagrunert

Reply:
It should work when #2270 is merged and I rebased on top of the master branch. I will verify it asap. :)

Reply:
The link works now as intended.